project-2-e-commerce/src/helpers/index.js

```diff
@@ -54,7 +54,7 @@ export const renderShoppingGuide = () => {
 }
 
 export const getTotal = (arr) => {
```

Comment:
```suggestion
export const getTotal = (arr = []) => {
```
Như thế này là dược rồi, k cần sửa đoạn bên trong đâu